### PR TITLE
optimize package import for stripe update

### PR DESCRIPTION
**Pull request by lcduong**
> **Short description of what this resolves:** import required dependencies
> 
> import required dependencies
> 
> **Changes proposed in this pull request:**
> 
> * import required dependencies
> 
> **Checklist**
> 
> * [x]   I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
> * [x]   My branch is up-to-date with the Upstream development branch.
> * [ ]   The acceptance, integration, unit tests and linter pass locally with my changes
> * [ ]   I have added tests that prove my fix is effective or that my feature works
> * [ ]   I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/pretix/helpers/database.py [hunk 1]
```diff
@@ -107,7 +107,9 @@ def as_sql(self, compiler, connection):
         params = lhs_params + rhs_params
         return '%s <> %s' % (lhs, rhs), params
 
+
 def _of_self():
     return ("self",) if connection.features.has_select_for_update_of else ()
 
+
```
Comment (mariobehling) [inline src/pretix/helpers/database.py]: The changes in that file are only empty lines.

File: src/pretix/helpers/http.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ def get_client_ip(request):
             ip = x_forwarded_for.split(',')[0]
     return ip
 
+
```
Comment (mariobehling) [inline src/pretix/helpers/http.py]: Also added empty line here.

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please avoid unneeded changes.

**Comment by lcduong** [comment]
Hi @mariobehling, I have removed empty  line and commited again, please review it

**Review by mariobehling — APPROVED** [review]

**Pull request merged**